### DISCUSSION TITLE
如何调用metaparse进行反射代码的生成？

### DISCUSSION THREAD

**Discussion by iamlivehaha** [BoomingTech/Piccolo#313] [Q&A]
![image](https://user-images.githubusercontent.com/30818771/180913598-2ddcc1ab-f1df-451c-8257-8791ac134b2c.png)

只在反射代码中找到下述描述，但看不太懂参数意义

**Comment by JCYTop** [comment]
同样想知道

**Comment by Ol6rin** [comment]
你好，我们的反射相关的文档还在准备中，目前知乎上有不错的文章可以先做参考。
https://zhuanlan.zhihu.com/p/493894595
